src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -278,13 +278,12 @@ protected DiagnosticGroups getDiagnosticGroups() {
   static DependencyOptions createDependencyOptions(
       boolean manageClosureDependencies,
       boolean onlyClosureDependencies,
-      boolean processCommonJSModules,
       List<String> closureEntryPoints)
```

Comment:
AFAIK this used to be a list of symbols (e.g. `foo.bar.myapp`), now it's a list of files, right? If so, would it make sense to change this to a `List<File>`, so we can also catch old code locations that work off incorrect assumptions?